src/shared/markdown.ts

```diff
@@ -336,7 +336,8 @@ export function getEmojiMart(
 }
 
 export async function setupTribute() {
-  if (Tribute === null) {
+  // eslint-disable-next-line eqeqeq
+  if (Tribute == null) {
```

Comment:
Was strict equals causing a bug?

Reply:
Yep, weirdly enough. It couldn't handle it as an import type, and I got a console error.